.github/scripts/common_utils.py

```diff
@@ -0,0 +1,104 @@
+import json
+import os
+import sys
+import time
+from github import Github, InputGitAuthor, GithubException
+
+ballerina_bot_username = os.environ['BALLERINA_BOT_USERNAME']
+ballerina_bot_token = os.environ['BALLERINA_BOT_TOKEN']
+ballerina_bot_email = os.environ['BALLERINA_BOT_EMAIL']
+ballerina_reviewer_bot_token = os.environ['BALLERINA_REVIEWER_BOT_TOKEN']
+github = Github(ballerina_bot_token)
+
+
+def update_md(bal_file_location, md_file, sample_name, sample_title):
+    with open(bal_file_location, 'r') as action_obj:
+        updated_bal_file = action_obj.read()
+    with open(md_file, 'a') as write_obj:
+        write_obj.write("```\n")
+        write_obj.write(updated_bal_file + "\n")
```

Comment:
We need to remove the finally added new line character `"\n"` in the md file. This will resolve [this issue](https://github.com/ballerina-platform/ballerina-dev-website/issues/6168)